client/my-sites/email/add-mailboxes/index.tsx

```diff
@@ -315,11 +345,16 @@ const AddMailboxes = ( props: AddMailboxesProps ): JSX.Element | null => {
 
 				<EmailHeader />
 
-				<HeaderCake onClick={ goToEmail }>
-					{ getTitanProductName() + ': ' + selectedDomainName }
-				</HeaderCake>
-				<MailboxNotices { ...additionalProps } mailProduct={ mailProduct } />
-				<MailboxesForm { ...additionalProps } goToEmail={ goToEmail } mailProduct={ mailProduct } />
+				<HeaderCake onClick={ goToEmail }>{ productName + ': ' + selectedDomainName }</HeaderCake>
+
+				<WithVerificationGate productFamily={ `${ productName }` } provider={ provider }>
```

Comment:
Couldn't we simplify this to the following?

```suggestion
				<WithVerificationGate productFamily={ productName } provider={ provider }>
```

Reply:
Product names are internationalised which are often represented as nodes, so they need to converted to a string in that case.

Reply:
I came to suggest that this issue can probably be addressed with types but apparently you took care of that in https://github.com/Automattic/wp-calypso/pull/64605/commits/cb7e525f4f4b652e31aeb735e453b654d725fdaf.